source/logHandler.py

```diff
@@ -78,7 +77,7 @@ def getCodePath(f):
 				if not member:
 					continue
 				memberType=type(member)
-				if memberType is FunctionType and member.func_code is f.f_code:

```

Comment:
I think I prefer something like `if isinstance(member, FunctionType)` while at it. Don't insist, though.